provider/aws/formation/rack.json

```diff
@@ -3228,13 +3228,80 @@
         }]
       }
     },
+    "CustomerManagedKMS": {
+      "DependsOn": ["ServiceRole", "SecureEnvironmentRole", "InstancesLifecycleRole", "InstancesLifecycleHandlerRole", "CustomTopicRole", "ApiRole"],
```

Comment:
You don't need to specify dependencies for resources you explicitly reference in other properties.
like `{ "Fn::GetAtt": ["ServiceRole", "Arn" ]},`